lib/block.ml

```diff
@@ -365,28 +368,42 @@ let mk ~line ~file ~column ~section ~labels ~legacy_labels ~header ~contents
       match header with
       | Some Header.OCaml ->
           let file_kind = Fk_ocaml { part_included = part } in
-          Ok (Include { file_included; file_kind })
+          Ok (`Include { file_included; file_kind })
       | _ ->
           check_not_set "`part` is not supported for non-OCaml code blocks."
             part
           >>= fun () ->
           let file_kind = Fk_other { header } in
-          Ok (Include { file_included; file_kind }) )
+          Ok (`Include { file_included; file_kind }) )
   | None -> (
       check_not_set "`part` label requires a `file` label." part >>= fun () ->
       match header with
       | Some Header.Shell ->
           check_no_errors errors >>= fun () ->
           check_not_set "`env` label cannot be used with a `shell` header." env
-          >>= fun () -> Ok (Cram { non_det })
+          >>= fun () -> Ok (`Cram { non_det })
```

Comment:
Any particular reason why we're using polymorphic variants again? If the name collide I think we should just scope them in a submodule or use different names for the kind alone and for the block themselves.

What do you think?

Reply:
Fixed.